README.md

```diff
@@ -18,6 +18,33 @@ For RISC-V compilation, you should take a compiler from here: https://www.embeco
 See [Getting Started with the Raspberry Pi Pico](https://rptl.io/pico-get-started) and the README in the [pico-sdk](https://github.com/raspberrypi/pico-sdk) for information
 on getting up and running.
 
+## Docker build and copy uf2 files back to host machine
+
+Build for a specific board by passing the board type as a build argument. Supported boards include:
+- pico
+- pico_w
+- pico2
+- pico2_w
+```bash
+# Create build directory
+mkdir -p build
+
+# Build Docker image (default is pico if --build-arg not specified)
+docker build -t pico-examples --build-arg PICO_BOARD=pico .
+
+# Run container for file copying
+docker run -d --name pico-build pico-examples sleep infinity
+
+# Copy UF2 files from container to host
+docker container exec pico-build mkdir -p /workspace/pico-examples/build/uf2files
+docker container exec pico-build find /workspace/pico-examples/build -name "*.uf2" -exec cp {} /workspace/pico-examples/build/uf2files \;
```

Comment:
Presumably this wouldn't work as expected if any UF2 files in different directories have the same filename? :wink: